test/redis/store/ttl_test.rb

```diff
@@ -110,6 +126,26 @@ class MockTtlStore < MockRedis
         redis.setnx(key, mock_value, options)
         redis.has_expire?(key, options[:expire_after]).must_equal true
       end
+
+      describe 'avoiding multi commands' do
+        let(:options) { { :expire_after => 3600, :avoid_multi_commands => true } }
+
+        it 'must call setnx with key and value and set raw to true' do
+          MockTtlStore.any_instance.expects(:multi).never
```

Comment:
It's best practice to stay away from `any_instance`, because it has the possibility of making the tests more volatile (that is, they fail randomly sometimes). Instead, we should think of a different way of mocking out this data. Why not expect that `redis` will call `pipelined` instead of expecting that any instance of `MockTtlStore` never calls `multi`?

Reply:
👍 I saw `any_instance` was being used in another test so I pushed an additional change just now to clean that up as well.